pkg/xds/envoy/listeners/v3/http_route_configurer.go

```diff
@@ -0,0 +1,75 @@
+package v3
+
+import (
+	envoy_core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	envoy_listener "github.com/envoyproxy/go-control-plane/envoy/config/listener/v3"
+	envoy_route "github.com/envoyproxy/go-control-plane/envoy/config/route/v3"
+	envoy_hcm "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/http_connection_manager/v3"
+	"github.com/pkg/errors"
+
+	envoy_routes "github.com/kumahq/kuma/pkg/xds/envoy/routes"
+)
+
+// HttpStaticRouteConfigurer configures a static set of routes into the
+// HttpConnectionManager in the filter chain.
+type HttpStaticRouteConfigurer struct {
+	Builder *envoy_routes.RouteConfigurationBuilder
+}
+
+var _ FilterChainConfigurer = &HttpStaticRouteConfigurer{}
+
+// Configure ...
```

Comment:
Can we put a more informed comment or just drop the comment at all if it's obvious?

Reply:
yeh this is a habit because my editor just runs revive and it complains in the default config. I'll remove it.